README.md

```diff
@@ -254,36 +257,61 @@ For more complex integrations, you may pass a `getLicense` function to fully con
 }
 ```
 
+
+#### Get Certificate by function
```

Comment:
I split the options into their own sections and added a mention of the widevine warning.